camelConnector/build.gradle

```
@@ -12,6 +12,8 @@ plugins {
 mainClassName = 'io.vantiq.extsrc.camelconn.connector.CamelMain'
 
 startScripts{
+    // Don't neeed all the class path stuff here since we've injected it into th main jar's manifest
```

typo:  e migrated from "the" ...
```suggestion
    // Don't need all the class path stuff here since we've injected it into the main jar's manifest
```